src/Endpoint/ConfigCreate.php

```diff
@@ -5,33 +5,33 @@
 class ConfigCreate extends \Docker\API\Runtime\Client\BaseEndpoint implements \Docker\API\Runtime\Client\Endpoint
 {
     /**
+     * 
      *
-     *
-     * @param null|\Docker\API\Model\ConfigsCreatePostBody $requestBody
+     * @param null|\Docker\API\Model\ConfigsCreatePostBody $requestBody 
      */
     public function __construct(?\Docker\API\Model\ConfigsCreatePostBody $requestBody = null)
     {
         $this->body = $requestBody;
     }
     use \Docker\API\Runtime\Client\EndpointTrait;
-    public function getMethod() : string
+    public function getMethod(): string
     {
         return 'POST';
     }
-    public function getUri() : string
+    public function getUri(): string
     {
         return '/configs/create';
     }
-    public function getBody(\Symfony\Component\Serializer\SerializerInterface $serializer, $streamFactory = null) : array
+    public function getBody(\Symfony\Component\Serializer\SerializerInterface $serializer, $streamFactory = null): array
```

Comment:
are these changes all automatic? e.g. would there be a way to get a smaller diff if we didn't change stuff like `:` vs ` :`?